xds/src/test/java/io/grpc/xds/RingHashLoadBalancerTest.java

```diff
@@ -1112,6 +1075,17 @@ public void nameResolutionErrorWithActiveSubchannels() {
     verifyNoMoreInteractions(helper);
   }
 
+  @Test
+  public void duplicateAddresses() {
+    RingHashConfig config = new RingHashConfig(10, 100);
+    List<EquivalentAddressGroup> servers =
+        createRepeatedServerAddrs(1, 2, 3);
+    boolean addressesAccepted = loadBalancer.acceptResolvedAddresses(
```

Comment:
Since this call has a side effect of updating LB state (see `handleNameResolutionError`) - would it make sense to test the state changed as expected?

Reply:
done

Reply:
Thanks for doing this!